taxi/views.py

```diff
@@ -87,3 +99,23 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
```

Comment:
The DriverDeleteView is correctly set up to delete a driver and redirect to the driver list page. Ensure that the corresponding button is available on the driver detail page as per the task requirements.